adders/paraglide/config/options.ts

```diff
@@ -0,0 +1,49 @@
+import { defineAdderOptions } from '@svelte-add/core';
+
+export const options = defineAdderOptions({
+	availableLanguageTags: {
+		question: 'Which language tags would you like to support?',
+		type: 'string',
+		default: '',
+		placeholder: 'en, de-ch',
+		validate(input) {
+			const { invalidLanguageTags, validLanguageTags } = parseLanguageTagInput(input);
+
+			if (invalidLanguageTags.length > 0)
+				return `Please enter a series of valid BCP47 language tags. Eg: en, de-ch, ar`;
```

Comment:
*if* this question is kept, this message should print the codes that are invalid so the user knows where the error is..